addons/hr_contract/report/hr_contract_history_report_views.xml

```diff
@@ -182,7 +182,7 @@
                                         <field name="display_name"/>
                                     </strong>
                                 </div>
-                                <div class="text-muted o_kanban_record_subtitle o_hr_contract_job_id">
+                                <div t-attf-class="text-muted o_kanban_record_subtitle o_hr_contract_job_id">
```

Comment:
commit 50a8028e66fa50169245330fdc9a43d72b25d993
Unnecessary change since you're not using any conditional in this class attribute